azext_iot/tests/central/__init__.py

```diff
@@ -43,21 +43,56 @@
 STORAGE_ACCOUNT = (
     settings.env.azext_iot_teststorageaccount or "iotstore" + generate_generic_id()[:4]
 )
+DEFAULT_FILE_UPLOAD_TTL = "PT1H"
 
 # Device templates
 device_template_path = get_context_path(__file__, "json/device_template_int_test.json")
-device_template_path_preview = get_context_path(
-    __file__, "json/device_template_int_test_preview.json"
-)
 edge_template_path_preview = get_context_path(
     __file__, "json/device_template_edge.json"
 )
 sync_command_params = get_context_path(__file__, "json/sync_command_args.json")
-device_attestation_path = get_context_path(__file__, "json/device_attestation.json")
-device_attestation2_path = get_context_path(__file__, "json/device_attestation2.json")
 device_updated_properties_path = get_context_path(__file__, "json/device_update_properties.json")
 device_updated_component_properties_path = get_context_path(__file__, "json/device_update_component_properties.json")
-DEFAULT_FILE_UPLOAD_TTL = "PT1H"
+
+# Device attestation
+attestation_payload = {
+    'type': 'symmetricKey',
+    'symmetricKey': {
+        # [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="int test only")]
+        'primaryKey': 'ya9+G4ED+/g0BgLduhjETJnbeEWMl1HIUApWCCpGMAU=',
+        # [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="int test only")]
+        'secondaryKey': 'D8jeWazxcCK+MAxrn9KlqDLb8trbuKs35KEbcLBnS48='
+    }
+}
+
+attestation_payload_update = {
+    'type': 'symmetricKey',
+    'symmetricKey': {
+        # [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="int test only")]
+        'primaryKey': 'r9KdK+LBaLiZ0p+RfAVj6eu9umGE6VqJj+AMLHdw+io=',
+        # [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="int test only")]
+        'secondaryKey': 'W7/4/oaMpA83RYfirH9vzic4ZeK2Piy0jO5rTOM5wxg='
+    }
+}
+
+scheduled_job_schedule = {
+    "start": "3022-05-24T22:29:01Z",
+    "recurrence": "daily"
+}
+
+scheduled_job_schedule_update = {
+    "start": "3022-05-24T22:29:01Z",
+    "recurrence": "daily"
+}
+
+scheduled_job_data = [
+    {
+        "type": "property",
+        "target": "",
+        "path": "testFirstProperty",
+        "value": "updated value"
+    }
+]
 
 
```

Comment:
Do any Central tests here depend on a Central app managed identity? Our nightly min CLI builds have been failing Central tests since that flag introduction because the extension supports min CLI core version of 2.30.0 and that managed identity flag was added in a later version of CLI core.

![image](https://user-images.githubusercontent.com/28658112/188023116-110708b0-8bb9-4713-b291-e87aadfb9ef8.png)

So if its not used we should remove creating an app with managed identity, or workaround it, like skip managed identity 
app provisioning + managed identity tests if CLI core is too old.

Reply:
I have no knowledge of this part. @lucadruda @yang-hai-feng  Any ideas?

Reply:
I don't think any tests depend on app managed identity.

Reply:
Then lets remove that flag, resolving the nightly pipeline issue.

Reply:
I'm going to make a quick pr for this so it could make the release